plugins/portals-list.js

```diff
@@ -4,6 +4,8 @@
 // @version        0.4.2
 // @description    Display a sortable list of all visible portals with full details about the team, resonators, links, etc.
 
+/* global plugin,DEPLOY_RESONATOR,CAPTURE_PORTAL,COMPLETION_BONUS,DESTROY_RESONATOR,DESTROY_LINK,DESTROY_FIELD -- eslint */
```

Comment:
Nooo, don't define them like this.
Use `window.` prefix to reference constants.
Also you might want to use some `var thisPlugin = window.plugins.thisPlugin` instead of defining 'plugin' here.

Reply:
yeah, I don't like it...it's just to make the dog happy - this is no code!!!!!!